src/datasource/query-user.ts

```diff
@@ -0,0 +1,15 @@
+import { User } from 'entity/user';
+import { CustomError } from 'error/errors';
+import { getRepository } from 'typeorm';
+
+export const queryUser = async (id: number) => {
+  const user = await getRepository(User).findOne({
+    where: { id: id },
+  });
```

Comment:
Dá pra omitir o `where` quando as condições de busca são só os campos da entidade:

```suggestion
  const user = await getRepository(User).findOne({ id });
```